src/promptflow-evals/promptflow/evals/evaluate/_utils.py

```diff
@@ -90,43 +74,42 @@ def _log_metrics_and_instance_results(
     # Adding line_number as index column this is needed by UI to form link to individual instance run
     instance_results["line_number"] = instance_results.index.values
 
-    ev_run = EvalRun(
+    with EvalRun(

```

Comment:
Maybe `EvalRunUploader` is better name?

Reply:
EvalRun creates the dummy azure job under the hood, which is being operated as a run.

Reply:
@nick863 This happens only when target is not present right ?

Reply:
Yes, otherwise, we just take the Promptflow run and use it for logging.